src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -253,64 +253,64 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       contents.title = 'Please review your new Statement of the Case';
       contents.description = (
         <p>The Veterans Benefits Administration sent you a new Statement of the Case on {formattedSocDate}
-        because after completing the remand instructions from the Board, they couldn’t fully grant
+        because, after completing the remand instructions from the Board, they couldn’t fully grant
         your appeal.</p>
       );
       break;
     }
     case STATUS_TYPES.pendingHearingScheduling:
-      contents.title = 'You’re waiting for your hearing to be scheduled';
+      contents.title = 'You’re waiting for the Board to schedule your hearing';
       contents.description = (
-        <p>You requested a {getHearingType(details.type)} hearing on your Form 9. When your hearing is scheduled, you will
-        receive a notice in the mail at least 30 days before the hearing date.</p>
+        <p>You requested a {getHearingType(details.type)} hearing on your Form 9. When the Board schedules your hearing, you’ll
+        get a notice in the mail at least 30 days before the hearing date.</p>
       );
       break;
     case STATUS_TYPES.scheduledHearing: {
       const formattedDate = moment(details.date, 'YYYY-MM-DD').format('MMMM Do, YYYY');
-      contents.title = 'Your hearing has been scheduled';
+      contents.title = 'The Board has scheduled your hearing';
       contents.description = (
-        <p>Your {getHearingType(details.type)} hearing is scheduled for {formattedDate} at {details.location}. If you need to change this
-        date, please contact your Veteran Service Organization or representative as soon as
+        <p>The Board scheduled your {getHearingType(details.type)} hearing for {formattedDate} at {details.location}. If you need to change this
+        date, please contact your Veterans Service Organization or representative as soon as
         possible.</p>
       );
       break;
     }
     case STATUS_TYPES.onDocket:
       contents.title = 'Your appeal is waiting to be assigned to a judge';
       contents.description = (
-        <p>Your appeal is at the Board of Veterans’ Appeals waiting to be assigned to a Veterans
-        Law Judge. Staff at the Board will make sure that your case is complete, accurate, and
+        <p>Your appeal is at the Board of Veterans’ Appeals, waiting to be assigned to a Veterans
+        Law Judge. Staff at the Board will make sure your case is complete, accurate, and
         ready to be decided by a judge.</p>
       );
       break;
     case STATUS_TYPES.atVso:
-      contents.title = 'Your appeal is currently with your Veteran Service Organization';
+      contents.title = 'Your Veterans Service Organization is preparing your appeal';
```

Comment:
revert back to 
contents.title = 'Your appeal is currently with your Veteran Service Organization';